server.js

```diff
@@ -1,16 +1,31 @@
 const express = require("express");
 const cors = require("cors");
-
+const uuid = require("uuid");
+const bodyParser = require("body-parser");
```

Comment:
Why are you importing this library? It is not normally included in modern express apps: https://medium.com/@mmajdanski/express-body-parser-and-why-may-not-need-it-335803cd048c

Reply:
I didn't know about it , I was thinking that there is just json parser